test/schema.js

```diff
@@ -1,33 +1,11 @@
 var fs = require("fs");
-var spec = require("../spec/contract.spec.json");
 var Ajv = require("ajv");
 var assert = require("assert");
 var Schema = require("../index.js");
 
 var MetaCoin = require("./MetaCoin.json");
 
 describe("Schema", function() {
-  var validator;
-  var invalidSchemaError;
-
-  before("load schema library", function() {
-    var ajv = new Ajv();
-    try {
-      validator = ajv.compile(spec);
-    } catch (e) {
-      invalidSchemaError = e;
-    }
-  });
-
-  it("validates as json-schema", function() {
-    assert.ifError(invalidSchemaError);
-  });
-
-  it("validates a simple example", function() {
-    var valid = validator(MetaCoin);
-    assert.ifError(validator.errors);
-  });
-
```

Comment:
This tests are all pointless. The other tests will rightly explode if the schema's invalid